source/_posts/2023-01-30-php-core-roundup-9.md

```diff
@@ -0,0 +1,428 @@
+
+---
+title: 'PHP Core Roundup #9'
+layout: post
+tags:
+    - roundup
+author:
+  name: Ayesh Karunaratne
+  url: https://aye.sh
+published_at: 30 January 2023
+---
+
+The first for the year 2023, and the ninth in the series, this episode of PHP Core Roundup is full of exciting new developments in PHP and about PHP 8.2, the latest PHP version released last month! January was a busy month with many RFC proposals, discussions, votes, and some that are already included for the upcoming PHP 8.3 version that is scheduled for November 2023.
+
+> The PHP Foundation currently supports six part-time PHP contributors who work on maintenance and new features for PHP. Maintenance is not limited to fixing bugs, but also includes work to reduce technical debt, making life easier for everyone working on PHP. The contributors funded by the PHP Foundation collaborate with other contributors on code, documentation, and discussions.
+
+<br>
+We publish the posts on our website, and you can subscribe to a newsletter; You don’t necessarily have to be a PHP Foundation backer to follow _PHP Core Roundup_.
+
+{% include "newsletter.html" %}
+
+## PHP 8.2 Released
+
+PHP 8.2 is here! With type-system improvements, `readonly `classes, sensitive parameter redaction support, a new "`random`" extension, and several new features and improvements, PHP 8.2.0 was released on December 08th, followed by a bug-fix release 8.2.1 on January 04th.
+
+Over **110 people**, along with our six PHP Foundation members, helped shape PHP 8.2. **Thank you for your amazing efforts 🙏🏼💜**.
+
+[PHP 8.2 Release Announcement](https://www.php.net/releases/8.2/en.php) on [php.net](https://php.net) contains a summary of what’s new and changed in PHP 8.2. Detailed lists and guides are also available on [PHP.Watch: PHP 8.2](https://php.watch/versions/8.2) and [Stitcher.io](https://stitcher.io/blog/new-in-php-82).
+
+
+## PHP 8.3 RFCs and Discussions
+
+Discussions and proposals for new features and changes for the upcoming PHP version 8.3 were being made even before PHP 8.2.0 was released. January was a particularly eventful month with several proposals and discussions, including a few that went to a vote, and a few that are now implemented in the PHP master branch.
+
+Most of the proposed improvements are in improving existing PHP functionality, especially the new readonly properties/class and the random extension. There were also discussions and proposals about adding a new syntax similar to Swift to declare granular get/set access to properties, bring back the literal-string RFC, passing the calling scope to class magic methods, and more.
+
+Additionally, the [Deprecations for PHP 8.3 RFC](https://wiki.php.net/rfc/deprecations_php_8_3) is also under discussion on features that the community proposes to deprecate in PHP 8.3 and remove in PHP 9.0
```

Comment:
I wouldn't call this RFC under discussion, it's mainly a W.I.P. draft until this gets officially announced probably around May.

Reply:
This is marked resolved, but it appears the phrasing wasn't updated.